src/features/profile/memberProfile/MemberProfile.tsx

```diff
@@ -31,8 +37,37 @@ const general_data = {
     'I am a digital artist based in Paris, France. My work has been featured in various galleries in Paris and New York City. I love playing with the characteristics of light in all its forms, and I like to challenge the way color is normally perceived in nature. I use various tools to create my work, including Rhino for 3D modeling and and Maxwell for rendering, with other work done in Photoshop and Illustrator.',
 }
 
+const mockCardProps: TNFTCardProps = {
+  author: 'vanecha',
+  avatarSrc: avatar,
+  imageSrc: image,
+  onSale: true,
+  price: '222K',
+  currencyName: 'PSL',
+  title: 'Infinity I',
+  rarenessPercent: 75,
+}
+
+const categories_options: TOption[] = [
```

Comment:
Please use camelCase for variables - standard naming in javascript

Reply:
fixed